kubernetes/samples/charts/apache-webtier/README.md

```diff
@@ -54,6 +54,7 @@ The following table lists the configurable parameters of the Apache webtier char
 | `volumePath`                       | Docker volume path for apache webtier                         | ``                    |
 | `createRBAC`                       | Boolean indicating if RBAC resources should be created        | `true`                |
 | `httpNodePort`                     | NodePort to expose for http access                            | `30305`               |
+| `httpsNodePort`                    | NodePort to expose for https access                           | `30443`               |
```

Comment:
How do we handle certificate files for SSL?

Reply:
The default certificate files are in apache image. We can use volume to provide custom certificate files. Then edit apache ssl configuration file to reference the custom certificate files.